src/features/generic_analyzer.h

```diff
@@ -0,0 +1,72 @@
+#ifndef _READMINDS_FEATURES_FACE_H_
+#define _READMINDS_FEATURES_FACE_H_
+
+#include "mediapipe/framework/formats/landmark.pb.h"
+#include "mediapipe/framework/port/opencv_core_inc.h"
+
+const int NTOTAL_LANDMARKS = 468;
+
+const int ANCHOR_LANDMARKS[] = {
+    1, 4, 5, 195, 197, 6,
+};
+
+// F4 feature anchors, landmarks
+// 1 and 4 was not needed
+const int F4_ANCHORS[] = {
+    5, 195, 197, 6,
+};
```

Comment:
We may use the same anchor for all features, just as Fernando did in the paper. Perhaps using more landmarks may rise some values a little high, but also might give better results. We could change the normalizing factor if points from `1` to `6` give very high results.

Reply:
Firstly I've set this anchor points because they better approximate Fernando's (i was a little worried about the difference between the values we get and the original feature values), however once our job is to improve the features get different values is a consequence, so I think use the same points in this case is the best option.